docs/GoogleDrive-batchsink.md

```diff
@@ -43,6 +43,11 @@ Make sure that:
 OAuth2 client credentials can be generated on Google Cloud
 [Credentials Page](https://console.cloud.google.com/apis/credentials)
 
+**OAuth Method:** The method used to get OAuth access tokens. The oauth access token can be directly provided,
```

Comment:
what is `OAuthMethod`, can you give some examples about method?

Reply:
This can be used where user wants to use oauthAccessToken macro. Added the same in doc.

Reply:
The inclusion of the additional line appears to be redundant.

Just keep it like this in the doc:
```
The method used to get OAuth access tokens. Users have the option to either directly provide the OAuth access token or supply a client ID, client secret, and refresh token.
```